ToDo/ToDoTableViewController.swift

```diff
@@ -0,0 +1,40 @@
+//
+//  ToDoTableViewController.swift
+//  ToDo
+//
+//  Created by Xcho on 08.12.21.
+//
+
+import UIKit
+
+class ToDoTableViewController: UITableViewController {
+    
+    var toDos = [ToDo]()
+
+    override func viewDidLoad() {
+        super.viewDidLoad()
+        
+        if let savedToDos = ToDo.loadToDos() {
```

Comment:
էս կարաս մի տողով էլ գրես, savedToDos = To.Do.loadToDos() ?? ToDo.loadToDosSamples()
վաբշե պետք չի էլի գրքինը մեկին մեկ կոպիա անել, փորձի հասկանաս թե ինչ ա գրել, ու եթե ավելի լավ ձև կա գրելու տենց գրի էլի։